src/tools/roc_recv/main.cpp

```diff
@@ -25,6 +26,50 @@
 
 using namespace roc;
 
+namespace {
+
+bool set_multicast_iface(packet::Address& address, const char* miface) {
```

Comment:
This should be a method of Address I think.

Reply:
Or maybe this should be `address::parse_miface`. Because we probably should expect `[]` brackets for IPv6 addresses and in this case we need some code to parse them.

Reply:
Because this is a bit confusing:

```
roc-recv -vv -s rtp+rs8m:[ff03::1]:10001 -r rs8m:[ff03::1]:10002 --miface ::
```

I'd rather expect:

```
roc-recv -vv -s rtp+rs8m:[ff03::1]:10001 -r rs8m:[ff03::1]:10002 --miface [::]
```

Reply:
Done

Reply:
The remaining thing to discuss. We have the following log record:

```
16:02:29.828 [dbg] roc_netio: udp receiver: joined multicast group ff00:: at interface ::
```

That seems a little bit non-consistent with our entire IPv6 address formatting. I think we can use something similar to:

```
16:02:29.828 [dbg] roc_netio: udp receiver: joined multicast group for port %s
```

and rely on the address_to_str formatting.

Thoughts?

Reply:
> and rely on the address_to_str formatting.

Agree.

Reply:
Done